.github/ct.yaml

```diff
@@ -0,0 +1,12 @@
+remote: origin
+target-branch: main
+chart-repos:
+  - bitnami=https://charts.bitnami.com/bitnami
+  - opensearch=https://opensearch-project.github.io/helm-charts
+chart-dirs:
+  - charts
+check-version-increment: false
```

Comment:
remembering your comment on one of my PRs...don't you want this enabled?

Reply:
Enabled for now and will test how it evals as was a bit iffy with this given we dont have automated releasing yet either and tedious to bump each chart for the values.yaml. Once we move to a single chart instead would be less of an issue